components.css

```
@@ -0,0 +1,266 @@
+	cursor: pointer;
+	color: var(--text-color);
+}
+
+.btn-primary {
+	background-color: var(--blue-2);
+	color: black;
+}
+
+.btn-primary:hover {
+	background-color: var(--blue-3);
+}
+
+.btn-link {
+	background-color: var(--purple-2);
+	color: black;
+}
+
+.btn-link:hover {
+	background-color: var(--purple-3);
+}
+
+.btn-floating {
+	position: fixed;
+	z-index: 10;
+	box-shadow: var(--default-box-shadow);
+	border: 1px solid black;
+}
+
+.card {
+	margin: var(--default-margin);
+	border: var(--default-border);
+	border-radius: var(--default-border-radius);
+	box-shadow: var(--default-box-shadow);
+}
+
+.img {
+	width: 15rem;
+	height: 15rem;
+	margin: var(--default-margin) 0rem;
+	border-radius: var(--default-border-radius);
+	box-shadow: var(--default-box-shadow);
+}
+
+.img-round {
+	border-radius: 100%;
+}
+
+.img-row {
+	width: 100%;
+	height: 50%;
+}
+
+.img-col {
+	width: 50%;
+	height: 100%;
+}
+
+.img-row,
+.img-col {
+	margin: 0rem;
+	padding: 0rem;
+}
+
+.input-container {
+	width: 20rem;
+	height: 1rem;
+	display: flex;
+	align-items: center;
+	margin: var(--default-margin);
+	padding: var(--default-padding);
+}
+
+.input-label {
+	width: fit-content;
+	height: 1rem;
+	background-color: var(--label-color);
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	border: var(--default-border);
+	border-radius: var(--default-border-radius);
+	padding: var(--default-padding);
+	margin: var(--default-margin);
+}
+
+.input {
+	width: 15rem;
+	height: 1rem;
+	border: var(--default-border);
+	border-radius: var(--default-border-radius);
+	padding: var(--default-padding);
+	margin: var(--default-margin);
+	background-color: var(--input-color);
+	color: var(--text-color);
+	font-size: 1rem;
+}
+
+.select {
+	width: 25rem;
+	height: 1rem;
+	padding: var(--default-padding);
+	margin: var(--default-margin);
+}
+
+.list {
+	margin: 1rem 0rem;
+}
+
+.list-stacked {
+	margin: 2rem 0rem;
+}
+
+.list-stacked li {
+	padding: 1rem;
+	border: var(--default-border);
+	border-bottom: none;
+}
+
+.list-stacked li:last-child {
+	border-bottom: var(--default-border);
+}
+
+.list-link {
+	padding: var(--default-padding);
+	border-radius: var(--default-border-radius);
+}
+
+.list-link:hover {
+	cursor: pointer;
+	background-color: var(--list-link-color-on-hover);
+}
+
+.list-link-active {
+	background-color: var(--list-link-color-on-hover);
+}
+
+.modal {
+	width: 25rem;
+	position: absolute;
+	top: 5%;
+	left: 35%;
+	background-color: var(--modal-bg-color);
+	margin: var(--default-margin);
+	border: var(--default-border);
+	border-radius: var(--default-border-radius);
+	box-shadow: var(--default-box-shadow);
+	overflow-y: auto;
+	padding: var(--default-padding);
+	z-index: 50;
+	color: black;
+}
+
+.modal-center {
+	top: 25%;
+	left: 35%;
+}
+
+.modal-backdrop {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	background-color: var(--modal-backdrop-color);
+	z-index: 10;
+	opacity: 0.98;
```

here also
```suggestion
	opacity: var(--opacity-std);
```
you can also keep track of opacity too